### PR TITLE
[UIE-197] Switch storybook

### PR DESCRIPTION
**Pull request by aherbst-broad**
Adds a storybook for the switch component.

Can be viewed at https://65fc89c9335768720ff8605a-hrupmmswdr.chromatic.com/?path=/docs/packages-components-switch--docs

### CODE REVIEW

File: src/stories/packages/components/Switch.stories.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      control: 'text',
+      description: 'text for the "off" state',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Switch>;
+
+export const Primary: Story = {
```
Comment (cahrens) [inline src/stories/packages/components/Switch.stories.ts]: Our convention is "Example".

Comment (aherbst-broad) [reply src/stories/packages/components/Switch.stories.ts]: Done.

File: src/stories/packages/components/Switch.stories.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        defaultValue: { summary: 'false' },
+      },
+    },
+    onLabel: {
+      control: 'text',
+      description: 'text for the "on" state',
+    },
+    offLabel: {
+      control: 'text',
+      description: 'text for the "off" state',
```
Comment (cahrens) [inline src/stories/packages/components/Switch.stories.ts]: I suggest adding documentation of `onChange` similar to https://github.com/DataBiosphere/terra-ui/blob/9ecb41a1442af71cd753950ea25923ea3f7725d5/src/stories/packages/components/Modal.stories.tsx#L79

**Review by cahrens — APPROVED** [review]
Thank you! Approving based on https://65fc89c9335768720ff8605a-yolwaxbemc.chromatic.com/?path=/docs/packages-components-switch--docs.

**Review by mlilynolting — APPROVED** [review]

**Pull request merged**